modules/core/site.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
 /* swipe event handler */
-var swipe_event = function(el, callback, direction) {
+var swipe_event = function (el, callback, direction) {
```

Comment:
Mixing code reformating with other changes makes it difficult to read.
https://en.wikipedia.org/wiki/Atomic_commit